### ISSUE TITLE
New development

### ISSUE THREAD

**Issue by Ronald206999** [github/docs#7519]
## Pre-requisites

- [ ] Prior to submitting documentation, please apply to join the GitHub Technology Partner Program: [partner.github.com/apply](https://partner.github.com/apply?partnershipType=Technology+Partner). Please feel free to proceed once your application is approved.

## Tasks

Please be sure to complete each of the following:

**Third-party product documentation:**

- [ ] MUST follow our [general contributing guidelines](CONTRIBUTING.md) for voice and markup format.
- [ ] MUST emphasize how the third-party product works with GitHub.
- [ ] MUST be written in Markdown format, using [one of the templates provided](contributing/github-partners/README.md#templates)
- [ ] MUST include the name and URL of the GitHub technology partner responsible for maintenance of the documentation being contributed. This should be added via the `contributor.name` and `contributor.URL` properties in the template's YAML frontmatter.
- [ ] MUST be proposed via a pull request to this repo following [the GitHub Flow](https://guides.github.com/introduction/flow/).
- [ ] MUST be located in the root of [the `content` folder](content). Your filename MUST match the GitHub technology partner name, and use the `.md` file extension.

**The `Pull Request`:**

- [ ] MUST reference this issue, e.g. via `closes #<this issue number>`
- [ ] MUST pass the automated CI checks
- [ ] MUST include links to supporting material demonstrating the functionality being documented (this can be a link to a public GitHub repo, _or_ a video / screencast walkthrough)

Once all tasks are completed, please mention `@github/docs-content` for next steps.

/cc @github/partner-engineering for :eyes:https://github.com/Ronald206999/credit/issues/4[1.1 bb763d35677c62f9f5d9728bba884662.cloudfront.net (CloudFront)]https://github.com/Ronald206999/credit/issues/4[3xgUE1cXEJhPK6ph7vCRuryiDrfzl5uPiYLF27IgsZoI4afWysA2vg==][AWSELB=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/, AWSELBCORS=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/;SECURE;SAMESITE=None][AWSELB=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/, AWSELBCORS=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/;SECURE;SAMESITE=None]https://crowncastle.comContent-Type[GET,PUT,POST,DELETE,OPTIONS][*][HTTP/1.1 200 OK][private, no-cache="set-cookie"][keep-alive][text/html; charset=utf-8][Microsoft-IIS/8.0][Accept-Encoding]Content-Type[HTTP/1.1 200 OK][AWSELB=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/, AWSELBCORS=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/;SECURE;SAMESITE=None][Microsoft-IIS/8.0][1.1 bb763d35677c62f9f5d9728bba884662.cloudfront.net (CloudFront)][3xgUE1cXEJhPK6ph7vCRuryiDrfzl5uPiYLF27IgsZoI4afWysA2vg==][SEA19-C3[4.0.30319][Miss from cloudfront]-][NETWORK 200][http/1.1]-bigteninch206999@gmail.com-juneauempire@gmail.com-valid certificate Ronald Lyle Juneau 3606892825Request Id: 1f4cb0dd-c1c3-43de-bd99-8f2c313f3c00
Correlation Id: 4c1b8507-3d15-473a-bc85-5a36099298c2
Timestamp: 2021-06-19T22:21:33Z
Message: AADSTS50020: User account 'juneauempire2020@gmail.com' from identity provider 'live.com' does not exist in tenant 'Microsoft Services' and cannot access the application 'f44b1140-bc5e-48c6-8dc0-5cf5a53c0e34'(Microsoft Edge) in that tenant. The account needs to be added as an external user in the tenant first. Sign out and sign in again with a different Azure Active Directory user account.Request Id: 1a6e4427-796d-4ea6-8599-194fc0d63300
Correlation Id: 4c1b8507-3d15-473a-bc85-5a36099298c2
Timestamp: 2021-06-19T22:25:35Z
Message: AADSTS50020: User account 'bigteninch206999@gmail.com' from identity provider 'live.com' does not exist in tenant 'Microsoft Services' and cannot access the application 'f44b1140-bc5e-48c6-8dc0-5cf5a53c0e34'(Microsoft Edge) in that tenant. The account needs to be added as an external user in the tenant first. Sign out and sign in again with a different Azure Active Directory user account.[Miss from cloudfront][4.0.30319][Accept-Encoding]Request Id: 1a6e4427-796d-4ea6-8599-194fc0d63300
Correlation Id: 4c1b8507-3d15-473a-bc85-5a36099298c2
Timestamp: 2021-06-19T22:25:35Z
Message: AADSTS50020: User account 'bigteninch206999@gmail.com' from identity provider 'live.com' does not exist in tenant 'Microsoft Services' and cannot access the application 'f44b1140-bc5e-48c6-8dc0-5cf5a53c0e34'(Microsoft Edge) in that tenant. The account needs to be added as an external user in the tenant first. Sign out and sign in again with a different Azure ActiveRequest Id: 1f4cb0dd-c1c3-43de-bd99-8f2c313f3c00
Correlation Id: 4c1b8507-3d15-473a-bc85-5a36099298c2
Timestamp: 2021-06-19T22:21:33Z
Message: AADSTS50020: User account 'juneauempire2020@gmail.com' from identity provider 'live.com' does not exist in tenant 'Microsoft Services' and cannot access the application 'f44b1140-bc5e-48c6-8dc0-5cf5a53c0e34'(Microsoft Edge) in that tenant. The account needs to be added as an external user in the tenant first. Sign out and sign in again with a different Azure Active Directory user account.[1.1 bb763d35677c62f9f5d9728bba884662.cloudfront.net (CloudFront)] Directoryhttps://github.com/Ronald206999/credit/issues/4 user account.https://github.com/Ronald206999/credit/issues/4[1.1 bb763d35677c62f9f5d9728bba884662.cloudfront.net (CloudFront)]https://github.com/Ronald206999/credit/issues/4[3xgUE1cXEJhPK6ph7vCRuryiDrfzl5uPiYLF27IgsZoI4afWysA2vg==][AWSELB=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/, AWSELBCORS=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/;SECURE;SAMESITE=None][AWSELB=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/, AWSELBCORS=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/;SECURE;SAMESITE=None]https://crowncastle.comContent-Type[GET,PUT,POST,DELETE,OPTIONS][*][HTTP/1.1 200 OK][private, no-cache="set-cookie"][keep-alive][text/html; charset=utf-8][Microsoft-IIS/8.0][Accept-Encoding]Content-Type[HTTP/1.1 200 OK][AWSELB=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/, AWSELBCORS=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/;SECURE;SAMESITE=None][Microsoft-IIS/8.0][1.1 bb763d35677c62f9f5d9728bba884662.cloudfront.net (CloudFront)][3xgUE1cXEJhPK6ph7vCRuryiDrfzl5uPiYLF27IgsZoI4afWysA2vg==][SEA19-C3[4.0.30319][Miss from cloudfront]-][NETWORK 200][http/1.1]-bigteninch206999@gmail.com-juneauempire@gmail.com-valid certificate Ronald Lyle Juneau 3606892825Request Id: 1f4cb0dd-c1c3-43de-bd99-8f2c313f3c00
Correlation Id: 4c1b8507-3d15-473a-bc85-5a36099298c2
Timestamp: 2021-06-19T22:21:33Z
Message: AADSTS50020: User account 'juneauempire2020@gmail.com' from identity provider 'live.com' does not exist in tenant 'Microsoft Services' and cannot access the application 'f44b1140-bc5e-48c6-8dc0-5cf5a53c0e34'(Microsoft Edge) in that tenant. The account needs to be added as an external user in the tenant first. Sign out and sign in again with a different Azure Active Directory user account.Request Id: 1a6e4427-796d-4ea6-8599-194fc0d63300
Correlation Id: 4c1b8507-3d15-473a-bc85-5a36099298c2
Timestamp: 2021-06-19T22:25:35Z
Message: AADSTS50020: User account 'bigteninch206999@gmail.com' from identity provider 'live.com' does not exist in tenant 'Microsoft Services' and cannot access the application 'f44b1140-bc5e-48c6-8dc0-5cf5a53c0e34'(Microsoft Edge) in that tenant. The account needs to be added as an external user in the tenant first. Sign out and sign in again with a different Azure Active Directory user account.[Miss from cloudfront][4.0.30319][Accept-Encoding][AWSELB=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/, AWSELBCORS=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/;SECURE;SAMESITE=None]https://crowncastle.comContent-Type[GET,PUT,POST,DELETE,OPTIONS][*][HTTP/1.1 200 OK][private, no-cache="set-cookie"][keep-alive][text/html; charset=utf-8][Microsoft-IIS/8.0][Accept-Encoding]Content-Type[HTTP/1.1 200 OK][AWSELB=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/, AWSELBCORS=0D7B637F1CE6F89A014CC0830DB91BBB15E697198029E087AB6EFE46FEE1E1257682797DEBE5DD51089C9A597FA1A4C648F13C1A52DFB6C9125E932C1C6C8BA3ABCB8C9C96;PATH=/;SECURE;SAMESITE=None][Microsoft-IIS/8.0][1.1 bb763d35677c62f9f5d9728bba884662.cloudfront.net (CloudFront)][3xgUE1cXEJhPK6ph7vCRuryiDrfzl5uPiYLF27IgsZoI4afWysA2vg==][SEA19-C3[4.0.30319][Miss from cloudfront]-][NETWORK 200][http/1.1]-bigteninch206999@gmail.com-juneauempire@gmail.com-valid certificate Ronald - - > Lyle Juneau 3606892825 valid certificate licences

**Comment by docubot** [comment]
This issue appears to have been opened accidentally. I'm going to close it now, but feel free to open a new issue or ask any questions in [discussions](https://github.com/github/docs/discussions)!
